tests/TailwindBuilderTest.php

```diff
@@ -33,6 +31,18 @@ protected function tearDown(): void
         foreach ($finder as $file) {
             unlink($file->getRealPath());
         }
+        $fs = new Filesystem();
+        if (file_exists(__DIR__.'/fixtures/var/tailwind')) {
```

Comment:
We can get rid of this check. The directory is always created in the `setup` method. Unless I'm missing something

Reply:
Indeed, done. I also reuse filesystem object to delete files instead of calling `unlink()` directly